sample.env

```diff
@@ -13,5 +13,7 @@ MONGODB_URI='mongodb://localhost:27017/fiddles'
 
 
 # PRIVATE section:-
+SESSION_SECRET='&E)H@McQfTjWnZr4'
```

Comment:
Maybe change this to something like `keyboard-cat` as an example this looks to be more like a password that someone is actually using.

Reply:
@joshghent Please can you take a look now.